examples/README.md

```diff
@@ -107,6 +107,16 @@ We encourage users to leverage `InProcessTransport` as demonstrated in the examp
 write unit tests. `InProcessTransport` is light-weight and runs the server
 and client in the same process without any socket/TCP connection.
 
+Mocking the client stub provides a false sense of security when writing tests. Mocking stubs and responses 
+allows for tests that don't map to reality, causing the tests to pass, but the system-under-test to fail. 
```

Comment:
I would also mention that it's very hard to accurately reproduce the reality by mocks, due to the complexity of the gRPC client library.

Reply:
Fixed